contracts/src/CrossChainApplications/examples/NativeTokenBridge/NativeTokenSource.sol

```diff
@@ -32,10 +32,9 @@ contract NativeTokenSource is
     ITokenSource,
     ReentrancyGuard
 {
-    // The address where the burned transaction fees are credited.
-    // Defined as BLACKHOLE_ADDRESS at
-    // https://github.com/ava-labs/subnet-evm/blob/e23ab058d039ff9c8469c89b139d21d52c4bd283/constants/constants.go
-    address public constant BURNED_TX_FEES_ADDRESS = 0x0100000000000000000000000000000000000000;
+    // Designated Blackhole Address for this contract. Tokens are sent here to be "burned" when
+    // a SourceAction.Burn message is received from the destination chain.
+    address public constant BURN_ADDRESS = 0x0100000000000000000000000000000000010203;
```

Comment:
how is this new burn address chosen? If it's not by random can you add in comment.

Reply:
It's just arbitrary

Reply:
Addresses of the form `0x010000000000000000000000000000000000XXXX` are reserved for precompiles.